integration_tests/integration/barcode/createNewContact.spec.ts

```diff
@@ -19,6 +19,14 @@ context('Create New Contact Page', () => {
     Page.verifyOnPage(ReviewRecipientsPage)
   })
 
+  it.only('should find a prison when searching with HMP prefix', () => {
```

Comment:
What does the `it.only` do?

Reply:
Pants. It only runs that test locally. I'll remove, good spot!